modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -1381,6 +1381,11 @@ def stack(self, level, dropna):
             pandas.to_numeric(df.squeeze(axis=1), *args, **kwargs)
         )
     )
+    to_timedelta = Map.register(
```

Comment:
I think we can pass `dtypes="timedelta64[ns]"` because the pandas docs say that the output is "Series of timedelta64 dtype". I can't find any `timedelta64` type besides `timedelta64[ns]`, so I guess that's the type to pass. Setting dtypes ahead of time means that we don't have to figure them out from the partitions later.